source/client/index.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
-import { render } from 'react-dom';
-import createApp from 'shared/components/app';
+import reactDom from 'react-dom/server';
+import { RoutingContext } from 'react-router';
 
-const App = createApp(React);
-const props = window.payload || {
-  title: 'Default client title'
+import { Provider } from 'react-redux';
+import configureStore from 'shared/configureStore';
+
+const render = reactDom.renderToString;
+
+const store = configureStore(window.__INITIAL_STATE__);
```

Comment:
These underscores are causing lint failures. Maybe just give it a more semantic name, instead? Maybe...

```js
const store = configureStore(window.BOOTSTRAP_CLIENT_STATE);
```

Remember that the server code needs to be updated as well.